@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1667,4 +1673,133 @@ export class TileGeometryCreator {
             lineFadeFar
         };
     }
+
+    private addClippingPlanes(object: THREE.Object3D, tile: Tile, mapView: MapView) {
```

Comment:
Please move this code to projection interface and also add unit-test (e.g. especially the poles and date border need some tests/special handling)